includes/class-wpsms.php

```
@@ -137,22 +176,20 @@ private function setupBackgroundProcess()
      */
     private function loadTextDomain()
     {
-        // Compatibility with WordPress < 5.0
-        if (function_exists('determine_locale')) {
-            $locale = apply_filters('plugin_locale', determine_locale(), 'wp-sms');
-
-            unload_textdomain('wp-sms', true);
-            load_textdomain('wp-sms', WP_LANG_DIR . '/wp-sms-' . $locale . '.mo');
-        }
-
         load_plugin_textdomain('wp-sms', false, WP_SMS_DIR . 'resources/languages');
```

PHPStan (2.2.7)

[error] Constant WP_SMS_DIR not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)